firestore-translate-text/PREINSTALL.md

```diff
@@ -21,4 +21,6 @@ This extension uses other Firebase or Google Cloud Platform services which may h
 - Cloud Firestore
 - Cloud Functions
 
-When you use Firebase Extensions, you're only charged for the underlying resources that you use. A paid-tier billing plan is only required if the extension uses a service that requires a paid-tier plan, for example calling to a Google Cloud Platform API or making outbound network requests to non-Google services. All Firebase services offer a free tier of usage. [Learn more about Firebase billing.](https://firebase.google.com/pricing)
+When you use Firebase Extensions, you're only charged for the underlying resources that you use. Due to the services used by this extension, the Blaze plan is required. This extension’s Cloud functions will run on Node.js 10. You will be charged a small amount (less than $0.10) when you deploy this extension, including when you make configuration changes and apply future updates. [Read FAQs](https://firebase.google.com/support/faq#functions-pricing).
```

Comment:
cc @kevinthecheung 

New suggested language for the billing section:

To install an extension, your project must be on the Blaze (pay as you go) plan.

- You will be charged [around $0.01 per month](https://cloud.google.com/functions/pricing#pricing_examples) for each instance of this extension you install.
- This extension uses other Firebase and Google Cloud Platform services, which have associated charges if you exceed the service's free tier:
    - Cloud Translation API
    - Cloud Functions
    - Cloud Firestore